cmd/build.go

```diff
@@ -158,17 +158,46 @@ func runBuild(cmd *cobra.Command, _ []string, newClient ClientFactory) (err erro
 		cfg buildConfig
 		f   fn.Function
 	)
-	if cfg, err = newBuildConfig().Prompt(); err != nil { // gather values into a single instruction set
+
+	// Initialize config first
+	cfg = newBuildConfig()
+
+	// Create function object to check if initialized
+	if f, err = fn.NewFunction(cfg.Path); err != nil {
```

Comment:
Now that i look at it we have a duplicate call to `NewFunction`. We should review and see if optimalization can be done there. Thats beyond the scope of this issue so no worries!